### PR TITLE
Uses circular icons for aave/makerdao for consistency

### LINKED ISSUE #1275: Use consistent icons for defi projects
## Abstract

We should use consistent icons (similar size/shape/format) when presenting defi protocols in selects etc. For example, in the defi selector we have a wide text-based icon for aave but a "tile" icon for Maker.

Sidenote: we need to update the kraken icon as it's outdated (new icons are purple on white or white on purple).

I have compiled the media kits from Aave and MKR's sites and attached below. They include SVG and PNGs of a variety of icon formats. Let's pick those that match.

e.g.: here are two that are same format
Maker\Badgers\Badge_Maker.svg
Aave\SVG\Roun 1/2/3.svg

[Aave MKR MediaKits.zip](https://github.com/rotki/rotki/files/5027971/Aave.MKR.MediaKits.zip)

### PR DESCRIPTION
**Pull request by kelsos**
Closes #1275

### CODE REVIEW

File: frontend/app/src/components/defi/Borrowing.vue [hunk 1]
```diff
@@ -76,7 +76,7 @@
                   aspect-ratio="1"
                   contain
                   position="left"
-                  max-width="48px"
+                  max-width="26px"
                   max-height="24px"
```
Comment (isidorosp) [inline frontend/app/src/components/defi/Borrowing.vue:80]: Any particular reason these are 24x26 instead of 24x24 (or I guess 26x26)?

Comment (kelsos) [reply frontend/app/src/components/defi/Borrowing.vue:80]: Yes, the image was not displaying as a proper circle for some reason. Part of the edge was getting cut.

Comment (isidorosp) [reply frontend/app/src/components/defi/Borrowing.vue:80]: Ok!

**Review by LefterisJP — APPROVED** [review]
LGTM -- tested with accounts that have data to show.

**Pull request merged**